src/toast-notification/classes.scss

```diff
@@ -17,7 +17,7 @@
   align-items: center;
   flex-direction: column;
   justify-content: center;
-  z-index: 1070;
+  z-index: 99999;
```

Comment:
Should we have a variable for it?

Reply:
You mean SASS one? Where would you reuse it?

Reply:
I wanted to say in popover but then understood that it is set in React. So no need then.